src/inputs/dropdownButton/dropdownButton.jsx

```diff
@@ -161,6 +161,7 @@ function DropdownButton(props) {
             target={target}
             text={text}
             title={title}
+            text={text}
```

Comment:
Okay, I mean it this time!  Pretty sure!
There are two `text={text}`s now.  Existing one on Line 162 and now adding this one on Line 164, which seems unnecessary and incorrect.

Reply:
Fixed.

Reply:
Fixed.  Resolving.